### ISSUE TITLE
Problem with <dir> element rendering

### ISSUE THREAD

**Issue by annakijas1** [rism-digital/verovio#2997]
When we take a MusicXML file and transform it to an MEIXML file using the Verovio app it seems to strip away the `<rend>` details that would normally appear within the `<dir> `element.

For example: if we want to add a `<dir>` of "slow-to-fast" to appear in normal font above a measure we would encode it as: 
```
<dir xml:id="dlqxbal" place="above" staff="1" tstamp="4.000000" vgrp="200">
           <rend fontfam="FreeSerif" fontstyle="normal">(slow to fast)</rend></dir>
```
Previously this encoding would automatically be in the MEIXML file after transforming from the MusicXML. But now when we transform from MusicXML to MEIXML the encoding appears like this leaving out the rendering instructions:

`<dir xml:id="dlqxbal" place="above" staff="1" tstamp="4.000000" vgrp="200">(slow to fast)</dir>`

The MEI file with this specific issue is available at https://github.com/annakijas1/rebalancing-music-canon/blob/main/assets/mei/dun_violin_and_piano_sonata002.mei.xml. Other files that we transformed to MEI as of Aug. 12 (earlier ones are fine) are also experiencing this issue.

We are using the development version of Verovio to render incipits in this project (https://github.com/annakijas1/rebalancing-music-canon). OS is MacOS, browser is Chrome and Safari.

**Comment by lpugin** [comment]
As far as I can see, the issue is in the MusicXML import and not related to the recent changes in the `<dir>` rendering. This is what I get (in the Verovio editor) with the encoding below. The text appears `normal` and `bold` as expected.

<img width="564" alt="image" src="https://user-images.githubusercontent.com/689412/184473618-aeb2e436-46fd-4d0b-bfd1-c2b3c479405e.png">

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title>Symbol example</title>
            </titleStmt>
            <pubStmt>
                <date isodate="2022-08-12"/>
            </pubStmt>
            <seriesStmt>
                <title>Verovio test suite</title>
            </seriesStmt>
            <notesStmt>
                <annot>Verovio supports symbols within directives.</annot>
            </notesStmt>
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application version="3.5.0" label="2">
                    <name>Verovio</name>
                </application>
            </appInfo>
        </encodingDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef>
                        <staffGrp symbol="bracket">
                            <staffDef label="Violino I" n="1" lines="5" clef.shape="G" clef.line="2" key.sig="1f" />
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <measure n="1">
                            <staff n="1">
                                <layer n="1">
                                    <beam>
                                        <note dur="16" oct="4" pname="e" />
                                        <note dur="16" oct="4" pname="e" />
                                        <note dur="16" oct="4" pname="e" />
                                        <note dur="16" oct="4" pname="e" />
                                    </beam>
                                    <beam>
                                        <note dur="8" oct="4" pname="e" />
                                        <note dur="8" oct="4" pname="e" />
                                    </beam>
                                    <note dur="4" oct="4" pname="e" />
                                    <note dur="4" oct="4" pname="e" />
                                </layer>
                            </staff>
                            <dir tstamp="1" place="above">
                                <symbol glyph.auth="smufl" glyph.num="U+E7E3" color="red"></symbol>
                            </dir>
                            <dir xml:id="dlqxbal" place="above" staff="1" tstamp="4.000000" vgrp="200">
                                <rend fontfam="FreeSerif" fontstyle="normal" fontweight="bold">(slow to fast)</rend></dir>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

Would it be possible to narrow it down to a minimal MusicXML file for reproducing the issue? Thanks

**Comment by annakijas1** [comment]
@lpugin I tested this again and it is a MusicXML issue. We're using the most current version of MuseScore and are experiencing an issue when we export from MuseScore to MusicXML the information (rend element) about the font does not transfer to the MusicXML file. I will check the MuseScore forum if anyone else is experiencing this.

**Comment by annakijas1** [comment]
Here is an example of how the notation and text looks when opened in MuseScore (as .mscz file)
<img width="926" alt="Screen Shot 2022-08-15 at 12 05 19 PM" src="https://user-images.githubusercontent.com/6841932/184671865-81f47f4f-715d-435c-ad83-42f7d508997c.png">

Here is an excerpt of the MusicXML that exports from MuseScore of this file without font information for the staff text (search for slow to fast):

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE score-partwise PUBLIC "-//Recordare//DTD MusicXML 3.1 Partwise//EN" "http://www.musicxml.org/dtds/partwise.dtd">
<score-partwise version="3.1">
  <work>
    <work-title>Violin and Piano Sonata (Hero)</work-title>
    </work>
  <identification>
    <creator type="composer">Tan Dun</creator>
    <encoding>
      <software>MuseScore 3.6.2</software>
      <encoding-date>2022-08-12</encoding-date>
      <supports element="accidental" type="yes"/>
      <supports element="beam" type="yes"/>
      <supports element="print" attribute="new-page" type="yes" value="yes"/>
      <supports element="print" attribute="new-system" type="yes" value="yes"/>
      <supports element="stem" type="yes"/>
      </encoding>
    </identification>
  <defaults>
    <scaling>
      <millimeters>6.99912</millimeters>
      <tenths>40</tenths>
      </scaling>
    <page-layout>
      <page-height>1596.77</page-height>
      <page-width>1233.87</page-width>
      <page-margins type="even">
        <left-margin>85.725</left-margin>
        <right-margin>85.725</right-margin>
        <top-margin>85.725</top-margin>
        <bottom-margin>85.725</bottom-margin>
        </page-margins>
      <page-margins type="odd">
        <left-margin>85.725</left-margin>
        <right-margin>85.725</right-margin>
        <top-margin>85.725</top-margin>
        <bottom-margin>85.725</bottom-margin>
        </page-margins>
      </page-layout>
    <word-font font-family="FreeSerif" font-size="10"/>
    <lyric-font font-family="FreeSerif" font-size="10"/>
    </defaults>
  <credit page="1">
    <credit-type>title</credit-type>
    <credit-words default-x="616.935" default-y="1511.05" justify="center" valign="top" font-size="22">Violin and Piano Sonata (Hero)</credit-words>
    </credit>
  <credit page="1">
    <credit-type>subtitle</credit-type>
    <credit-words default-x="616.935" default-y="1453.9" justify="center" valign="top" font-size="16">II. Above Water</credit-words>
    </credit>
  <credit page="1">
    <credit-type>composer</credit-type>
    <credit-words default-x="1148.14" default-y="1411.05" justify="right" valign="bottom">Tan Dun</credit-words>
    </credit>
  <part-list>
    <score-part id="P1">
      <part-name>Violin</part-name>
      <part-abbreviation>Vln.</part-abbreviation>
      <score-instrument id="P1-I1">
        <instrument-name>Violin</instrument-name>
        </score-instrument>
      <midi-device id="P1-I1" port="1"></midi-device>
      <midi-instrument id="P1-I1">
        <midi-channel>1</midi-channel>
        <midi-program>41</midi-program>
        <volume>78.7402</volume>
        <pan>0</pan>
        </midi-instrument>
      </score-part>
    </part-list>
  <part id="P1">
    <measure number="1" width="608.41">
      <print>
        <system-layout>
          <system-margins>
            <left-margin>50.00</left-margin>
            <right-margin>0.00</right-margin>
            </system-margins>
          <top-system-distance>185.19</top-system-distance>
          </system-layout>
        </print>
      <attributes>
        <divisions>20</divisions>
        <key>
          <fifths>0</fifths>
          <mode>none</mode>
          </key>
        <time>
          <beats>4</beats>
          <beat-type>4</beat-type>
          </time>
        <clef>
          <sign>G</sign>
          <line>2</line>
          </clef>
        </attributes>
      <direction placement="above">
        <direction-type>
          <words default-y="9.37" relative-x="-57.00" relative-y="10.00">pizz.</words>
          </direction-type>
        </direction>
      <direction placement="above">
        <direction-type>
          <words default-x="-52.14" default-y="41.84" relative-y="20.00" font-weight="bold" font-size="12">Andante Molto </words>
          </direction-type>
        <direction-type>
          <metronome parentheses="yes" default-x="-52.14" default-y="41.84" relative-y="20.00">
            <beat-unit>quarter</beat-unit>
            <per-minute>60</per-minute>
            </metronome>
          </direction-type>
        <sound tempo="60"/>
        </direction>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="3.29" default-y="-40.00" relative-y="-25.00">
            <f/>
            </dynamics>
          </direction-type>
        <sound dynamics="106.67"/>
        </direction>
      <note default-x="95.40" default-y="20.00">
        <pitch>
          <step>C</step>
          <alter>1</alter>
          <octave>6</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <accidental>sharp</accidental>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">begin</beam>
        <beam number="2">begin</beam>
        <notations>
          <tuplet type="start" bracket="no"/>
          <slur type="start" placement="above" number="1"/>
          <articulations>
            <accent/>
            </articulations>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="15.42" relative-y="10.00">+</words>
          </direction-type>
        </direction>
      <note default-x="133.64" default-y="15.00">
        <pitch>
          <step>B</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="10.42" relative-y="10.00">+</words>
          </direction-type>
        </direction>
      <note default-x="171.88" default-y="10.00">
        <pitch>
          <step>A</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="5.42" relative-y="10.00">+</words>
          </direction-type>
        </direction>
      <note default-x="210.13" default-y="5.00">
        <pitch>
          <step>G</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-y="10.00">+</words>
          </direction-type>
        </direction>
      <note default-x="248.37" default-y="-5.00">
        <pitch>
          <step>E</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">end</beam>
        <beam number="2">end</beam>
        <notations>
          <tuplet type="stop"/>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-y="10.00">III</words>
          </direction-type>
        </direction>
      <note default-x="286.61" default-y="-35.00">
        <pitch>
          <step>F</step>
          <octave>4</octave>
          </pitch>
        <duration>10</duration>
        <voice>1</voice>
        <type>eighth</type>
        <accidental>natural</accidental>
        <stem>up</stem>
        <notations>
          <articulations>
            <accent/>
            </articulations>
          <slide line-type="solid" number="1" type="start" default-y="-34.34"/>
          </notations>
        </note>
      <note default-x="355.19" default-y="-30.00">
        <pitch>
          <step>G</step>
          <octave>4</octave>
          </pitch>
        <duration>20</duration>
        <voice>1</voice>
        <type>quarter</type>
        <stem>up</stem>
        <notations>
          <slur type="stop" number="1"/>
          <slide line-type="solid" number="1" type="stop"/>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="19.78" relative-x="10.00" relative-y="10.00">IV
</words>
          <words>arco</words>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="6.58" default-y="-90.00" relative-y="-25.00">
            <p/>
            </dynamics>
          </direction-type>
        <sound dynamics="54.44"/>
        </direction>
      <direction placement="below">
        <direction-type>
          <wedge type="crescendo" number="1" default-y="-110.00"/>
          </direction-type>
        </direction>
      <note default-x="446.71" default-y="-50.00">
        <pitch>
          <step>C</step>
          <octave>4</octave>
          </pitch>
        <duration>10</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <notations>
          <slur type="start" placement="below" number="1"/>
          <slide line-type="solid" number="1" type="start" default-y="-51.88"/>
          </notations>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1"/>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="6.58" default-y="-90.00" relative-y="-25.00">
            <fp/>
            </dynamics>
          </direction-type>
        <sound dynamics="106.67"/>
        </direction>
      <direction placement="above">
        <direction-type>
          <words relative-x="25.86" relative-y="40.05">(slow to fast)</words>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <wedge type="diminuendo" number="1" default-y="-110.00"/>
          </direction-type>
        </direction>
      <note default-x="515.28" default-y="-60.00">
        <pitch>
          <step>A</step>
          <octave>3</octave>
          </pitch>
        <duration>20</duration>
        <tie type="start"/>
        <voice>1</voice>
        <type>quarter</type>
        <stem>up</stem>
        <notations>
          <tied type="start"/>
          <slur type="stop" number="1"/>
          <fermata type="upright" default-y="33.49" relative-y="5.00"/>
          <articulations>
            <accent/>
            </articulations>
          <ornaments>
            <trill-mark/>
            <wavy-line type="start" number="1" default-y="12.82"/>
            </ornaments>
          <slide line-type="solid" number="1" type="stop"/>
          </notations>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1"/>
          </direction-type>
        </direction>
      </measure>
    <measure number="2" width="370.29">
      <direction placement="below">
        <direction-type>
          <wedge type="crescendo" number="1" default-y="-110.00"/>
          </direction-type>
        </direction>
      <note print-object="no">
        <rest/>
        <duration>80</duration>
        <voice>1</voice>
        <type>whole</type>
        <notations>
          <articulations>
            <breath-mark/>
            </articulations>
          </notations>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="2"/>
          </direction-type>
        </direction>
      <backup>
        <duration>80</duration>
        </backup>
      <note default-x="16.80" default-y="-60.00">
        <pitch>
          <step>A</step>
          <octave>3</octave>
          </pitch>
        <duration>10</duration>
        <tie type="stop"/>
        <voice>2</voice>
        <type>eighth</type>
        <stem>down</stem>
        <notations>
          <tied type="stop"/>
          </notations>
        </note>
      <note>
        <rest/>
        <duration>10</duration>
        <voice>2</voice>
        <type>eighth</type>
        </note>
      <note>
        <rest/>
        <duration>20</duration>
        <voice>2</voice>
        <type>quarter</type>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1"/>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <wedge type="diminuendo" number="2" default-y="-110.00"/>
          </direction-type>
        </direction>
      <note>
        <rest/>
        <duration>20</duration>
        <voice>2</voice>
        <type>quarter</type>
        <notations>
          <fermata type="upright" relative-y="-18.20"/>
          </notations>
        </note>
      <note>
        <rest/>
        <duration>20</duration>
        <voice>2</voice>
        <type>quarter</type>
        <notations>
          <ornaments>
            <wavy-line type="stop" number="1"/>
            </ornaments>
          </notations>
        </note>
      </measure>
    <measure number="3" width="425.67">
      <print new-system="yes">
        <system-layout>
          <system-margins>
            <left-margin>0.00</left-margin>
            <right-margin>0.00</right-margin>
            </system-margins>
          <system-distance>169.74</system-distance>
          </system-layout>
        </print>
      <note>
        <rest/>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <notations>
          <tuplet type="start" bracket="yes"/>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-x="-45.00" relative-y="26.00">pizz.</words>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="3.29" default-y="-52.94" relative-y="-25.00">
            <f/>
            </dynamics>
          </direction-type>
        <sound dynamics="106.67"/>
        </direction>
      <note default-x="86.21" default-y="15.00">
        <pitch>
          <step>B</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">begin</beam>
        <beam number="2">begin</beam>
        <notations>
          <slur type="start" placement="above" number="1"/>
          <articulations>
            <accent/>
            </articulations>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-y="21.66">+</words>
          </direction-type>
        </direction>
      <note default-x="113.17" default-y="10.00">
        <pitch>
          <step>A</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-y="13.87">+</words>
          </direction-type>
        </direction>
      <note default-x="144.47" default-y="0.00">
        <pitch>
          <step>F</step>
          <alter>1</alter>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <accidental>sharp</accidental>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-y="7.71">+</words>
          </direction-type>
        </direction>
      <note default-x="171.43" default-y="-5.00">
        <pitch>
          <step>E</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">end</beam>
        <beam number="2">end</beam>
        <notations>
          <tuplet type="stop"/>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="24.41" relative-y="10.00">II</words>
          </direction-type>
        </direction>
      <note default-x="198.38" default-y="-5.00">
        <pitch>
          <step>E</step>
          <octave>5</octave>
          </pitch>
        <duration>10</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>down</stem>
        <notations>
          <articulations>
            <accent/>
            </articulations>
          <slide line-type="solid" number="1" type="start" default-y="-6.88"/>
          </notations>
        </note>
      <note default-x="246.72" default-y="-15.00">
        <pitch>
          <step>C</step>
          <alter>1</alter>
          <octave>5</octave>
          </pitch>
        <duration>20</duration>
        <voice>1</voice>
        <type>quarter</type>
        <accidental>sharp</accidental>
        <stem>down</stem>
        <notations>
          <slur type="stop" number="1"/>
          <slide line-type="solid" number="1" type="stop"/>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="29.65" relative-x="10.00" relative-y="10.00">arco
</words>
          <words>IV
</words>
          <words></words>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="6.58" default-y="-75.00" relative-y="-25.00">
            <p/>
            </dynamics>
          </direction-type>
        <sound dynamics="54.44"/>
        </direction>
      <direction placement="below">
        <direction-type>
          <wedge type="crescendo" number="1" default-y="-95.00"/>
          </direction-type>
        </direction>
      <note default-x="311.23" default-y="-40.00">
        <pitch>
          <step>E</step>
          <octave>4</octave>
          </pitch>
        <duration>10</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <notations>
          <slur type="start" placement="below" number="1"/>
          <slide line-type="solid" number="1" type="start" default-y="-40.94"/>
          </notations>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1"/>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="6.58" default-y="-75.00" relative-y="-25.00">
            <fp/>
            </dynamics>
          </direction-type>
        <sound dynamics="106.67"/>
        </direction>
      <direction placement="above">
        <direction-type>
          <words relative-x="27.00" relative-y="48.00">(slow to fast)</words>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <wedge type="diminuendo" number="1" default-y="-95.00"/>
          </direction-type>
        </direction>
      <note default-x="359.56" default-y="-45.00">
        <pitch>
          <step>D</step>
          <octave>4</octave>
          </pitch>
        <duration>20</duration>
        <tie type="start"/>
        <voice>1</voice>
        <type>quarter</type>
        <stem>up</stem>
        <notations>
          <tied type="start"/>
          <slur type="stop" number="1"/>
          <fermata type="upright" default-y="33.49" relative-y="5.00"/>
          <articulations>
            <accent/>
            </articulations>
          <ornaments>
            <trill-mark/>
            <wavy-line type="start" number="1" default-y="12.82"/>
            </ornaments>
          <slide line-type="solid" number="1" type="stop"/>
          </notations>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1"/>
          </direction-type>
        </direction>
      </measure>
    <measure number="4" width="217.30">
      <direction placement="below">
        <direction-type>
          <wedge type="crescendo" number="1" default-y="-95.00"/>
          </direction-type>
        </direction>
      <note print-object="no">
        <rest/>
        <duration>80</duration>
        <voice>1</voice>
        <type>whole</type>
        <notations>
          <articulations>
            <breath-mark/>
            </articulations>
          </notations>
        </note>
      <backup>
        <duration>80</duration>
        </backup>
      <note default-x="13.00" default-y="-45.00">
        <pitch>
          <step>D</step>
          <octave>4</octave>
          </pitch>
        <duration>10</duration>
        <tie type="stop"/>
        <voice>2</voice>
        <type>eighth</type>
        <stem>down</stem>
        <notations>
          <tied type="stop"/>
          </notations>
        </note>
      <note>
        <rest/>
        <duration>10</duration>
        <voice>2</voice>
        <type>eighth</type>
        </note>
      <note>
        <rest/>
        <duration>20</duration>
        <voice>2</voice>
        <type>quarter</type>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1"/>
          </direction-type>
        </direction>
      <direction placement="below">
        <direction-type>
          <wedge type="diminuendo" number="1" default-y="-95.00"/>
          </direction-type>
        </direction>
      <note>
        <rest/>
        <duration>20</duration>
        <voice>2</voice>
        <type>quarter</type>
        <notations>
          <fermata type="upright" relative-y="-18.20"/>
          </notations>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="1" relative-x="51.25"/>
          </direction-type>
        </direction>
      <note>
        <rest/>
        <duration>20</duration>
        <voice>2</voice>
        <type>quarter</type>
        <notations>
          <ornaments>
            <wavy-line type="stop" number="1"/>
            </ornaments>
          </notations>
        </note>
      </measure>
    <measure number="5" width="385.73">
      <attributes>
        <time>
          <beats>3</beats>
          <beat-type>4</beat-type>
          </time>
        </attributes>
      <direction placement="below">
        <direction-type>
          <dynamics default-x="3.29" default-y="-40.00" relative-y="-25.00">
            <f/>
            </dynamics>
          </direction-type>
        <sound dynamics="106.67"/>
        </direction>
      <direction placement="above">
        <direction-type>
          <words default-y="46.11" relative-x="-10.00" relative-y="10.00">pizz.</words>
          </direction-type>
        </direction>
      <note default-x="35.30" default-y="15.00">
        <pitch>
          <step>B</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">begin</beam>
        <beam number="2">begin</beam>
        <notations>
          <tuplet type="start" bracket="no"/>
          <slur type="start" placement="above" number="1"/>
          <articulations>
            <accent/>
            </articulations>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words relative-y="22.88">+</words>
          </direction-type>
        </direction>
      <note default-x="61.12" default-y="10.00">
        <pitch>
          <step>A</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="3.89" relative-y="16.53">+</words>
          </direction-type>
        </direction>
      <note default-x="86.94" default-y="5.00">
        <pitch>
          <step>G</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="3.58" relative-y="12.74">+</words>
          </direction-type>
        </direction>
      <note default-x="116.64" default-y="0.00">
        <pitch>
          <step>F</step>
          <alter>1</alter>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <accidental>sharp</accidental>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">continue</beam>
        <beam number="2">continue</beam>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="2.60" relative-y="8.90">+</words>
          </direction-type>
        </direction>
      <note default-x="142.46" default-y="-5.00">
        <pitch>
          <step>E</step>
          <octave>5</octave>
          </pitch>
        <duration>4</duration>
        <voice>1</voice>
        <type>16th</type>
        <time-modification>
          <actual-notes>5</actual-notes>
          <normal-notes>4</normal-notes>
          </time-modification>
        <stem>down</stem>
        <beam number="1">end</beam>
        <beam number="2">end</beam>
        <notations>
          <tuplet type="stop"/>
          </notations>
        </note>
      <direction placement="above">
        <direction-type>
          <words default-y="40.09" relative-y="10.00">(only &quot;pizz&quot; the first note)</words>
          </direction-type>
        </direction>
      <note default-x="168.29" default-y="-45.00">
        <pitch>
          <step>D</step>
          <octave>4</octave>
          </pitch>
        <duration>5</duration>
        <voice>1</voice>
        <type>16th</type>
        <stem>up</stem>
        <beam number="1">begin</beam>
        <beam number="2">begin</beam>
        <notations>
          <slur type="start" placement="below" number="2"/>
          <articulations>
            <accent/>
            </articulations>
          </notations>
        </note>
      <note default-x="199.09" default-y="-40.00">
        <pitch>
          <step>E</step>
          <octave>4</octave>
          </pitch>
        <duration>5</duration>
        <voice>1</voice>
        <type>16th</type>
        <stem>up</stem>
        <beam number="1">continue</beam>
        <beam number="2">end</beam>
        <notations>
          <slide line-type="solid" number="1" type="start" default-y="-37.05"/>
          </notations>
        </note>
      <note default-x="229.90" default-y="-30.00">
        <pitch>
          <step>G</step>
          <octave>4</octave>
          </pitch>
        <duration>10</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <beam number="1">end</beam>
        <notations>
          <slur type="stop" number="1"/>
          <slur type="stop" number="2"/>
          <fermata type="upright" default-y="20.47" relative-y="5.00"/>
          <slide line-type="solid" number="1" type="stop"/>
          </notations>
        </note>
      <note default-x="276.21" default-y="-65.00">
        <pitch>
          <step>G</step>
          <octave>3</octave>
          </pitch>
        <duration>5</duration>
        <voice>1</voice>
        <type>16th</type>
        <stem>up</stem>
        <beam number="1">begin</beam>
        <beam number="2">begin</beam>
        <notations>
          <slur type="start" placement="below" number="1"/>
          <articulations>
            <accent/>
            </articulations>
          </notations>
        </note>
      <note default-x="307.02" default-y="-60.00">
        <pitch>
          <step>A</step>
          <octave>3</octave>
          </pitch>
        <duration>5</duration>
        <voice>1</voice>
        <type>16th</type>
        <stem>up</stem>
        <beam number="1">continue</beam>
        <beam number="2">end</beam>
        <notations>
          <slide line-type="solid" number="1" type="start" default-y="-55.82"/>
          </notations>
        </note>
      <note default-x="337.83" default-y="-50.00">
        <pitch>
          <step>C</step>
          <octave>4</octave>
          </pitch>
        <duration>10</duration>
        <voice>1</voice>
        <type>eighth</type>
        <stem>up</stem>
        <beam number="1">end</beam>
        <notations>
          <slur type="stop" number="1"/>
          <fermata type="upright" relative-y="5.00"/>
          <slide line-type="solid" number="1" type="stop"/>
          </notations>
        </note>
      </measure>
    </part>
  </score-partwise>`

```

**Comment by rettinghaus** [comment]
This is a known problem, see #2045. 
TLDR by default MuseScore does not include font information, Verovio's default style for `dir` is italics.

You could override Verovio's default style with CSS:

```css
.dir text {
    font-style:normal 
}
```

**Comment by annakijas1** [comment]
Thank you @rettinghaus. This has fixed the problem for us!

**Comment by annakijas1** [comment]
I’ll take a closer look at the MusicXML file to see if that is where the issue is. We have been exporting the XML from MuseScore and transforming the XML to MEI using the MEI friend app or Verovio Editor and the font style in the render element always transferred over until the other day.

**Comment by lpugin** [comment]
@annakijas1 I think this must be due to the changes made to support symbols mixed with text introduced in the latest release. I will have look

**Comment by lpugin** [comment]
It seems to work for me

![image](https://user-images.githubusercontent.com/689412/195054522-3f95200a-7fd0-4fa8-8983-798e1a8519a3.png)

This is loading [this](https://github.com/rism-digital/verovio.org/blob/gh-pages/_tests/symbol/symbol-002.mei) file with the VerovioApp. I would need more input from your side. Do you have an example file that is not showing the symbols anymore?
